nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -436,9 +436,6 @@ static void phNciNfc_ResetNtfDelayCb(uint32_t dwTimerId, void *pContext)
     if (NULL != pNciContext)
     {
         (void)phOsalNfc_Timer_Stop(pNciContext->dwNtfTimerId);
-        (void)phOsalNfc_Timer_Delete(pNciContext->dwNtfTimerId);
-
-        pNciContext->dwNtfTimerId = 0;
```

Comment:
Apparently the error you're hitting is occurring after a sequence of events:
1) pNciContext->dwNtfTimerId was created in phNciNfc_Discovery.c or phNciNfc_Init.c
2) New timer instance was created in one of those files and has been reassigned to pNciContext->dwNtfTimerId, without checking and cleaning up previously assigned value.
3) phOsalNfc_Timer_Stop and/or phOsalNfc_Timer_Delete is called for each of these created instances, but second one is failing.

One variable is used for two different purposes, and usage time intersects. Fix me if I'm wrong. But if the assumption is correct, than these changes are only masking the problem. As a suggestion, can you verify that pNciContext->dwNtfTimerId is not a valid timer before "pNciContext->dwNtfTimerId = phOsalNfc_Timer_Create();" reassignment? Using the same scenario you used to hit those timer errors.